test/integration/api-support/test/index.test.js

```diff
@@ -53,6 +53,13 @@ function runTests (serverless = false) {
     expect(json).toEqual({ error: 'Server error!' })
   })
 
+  it('should throw Internal Server Error', async () => {
+    const res = await fetchViaHTTP(appPort, '/api/user-error', null, {})
+    const text = await res.text()
+    expect(res.status).toBe(500)
```

Comment:
I don't think the status would be 500 in that case? Is anything setting it to 500?